uwsgitop

```diff
@@ -259,7 +259,7 @@ def main():
             for worker in dd['workers']:
                 sigs = 0
                 wtx = human_size(worker['tx'])
-                wlastspawn = "--:--:--"
+                wlastspawn = "-"
```

Comment:
open to suggestions regarding what to put here.

Reply:
I think this should be always set so no need to change it